webapp/src/routes/BPJson.js

```diff
@@ -44,14 +44,16 @@ const getBPJsonUrl = async (producer = {}) => {
       data: { chains }
     } = await axios.get(chainsUrl)
     chainUrl = chains[ualConfig.network.chainId] || chainUrl
-  } catch (error) {}
+  } catch (error) { }
 
   return `${producerUrl}/${chainUrl}`.replace(/(?<=:\/\/.*)((\/\/))/, '/')
 }
 
 const getBpJSONOffChain = async (producer) => {
   try {
     const bpUrl = await getBPJsonUrl(producer)
+
+    if (bpUrl === 'http:///bp.json') return undefined
```

Comment:
how are we handling bp.json files on sites without SSL?

Reply:
We make the request with http and not with https.

Reply:
![image](https://user-images.githubusercontent.com/5632966/166973749-eb59dc6b-14cf-40f7-bf2f-60c1ed0a2f46.png)
[kubernetes/configmap-dashboard.yaml](https://github.com/eoscostarica/eosio-dashboard/pull/780/files/a5d7d2939478cf9940e1b430fea4cbeabc3aed5e#diff-7af4d6ed28e83b4e4e50624862513b116f45d4cda453c742cc4031997532602f)